lib/cuckoo/common/trim_utils.py

```diff
@@ -0,0 +1,56 @@
+from contextlib import suppress
+
+import olefile
+
+from lib.cuckoo.common.config import Config
+from lib.cuckoo.common.objects import File
+from lib.cuckoo.common.integrations.parse_pe import PortableExecutable
+from lib.cuckoo.common.path_utils import path_write_file
+
+web_cfg = Config("web")
+
+
+def trim_file(filename: bytes, options: str, doc: bool = False, chunk: bytes=False, return_size:bool=False) -> bool:
```

Comment:
I don't see where the 'chunk' or 'return_size' parameters ever get passed in.

Reply:
looks like this is still work in progress - i have some updates that I will push later for review